packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -34,6 +36,8 @@ class TwinDeploymentHandler {
       }
     }
     try {
+      const kycStatus = await this.kyc.status();
+      if (kycStatus !== KycStatus.verified) throw new ValidationError("KYC is not verified.");
```

Comment:
might be better as "Verification Failed", user doesn't necessarily know what KYC is.